client/src/pages/Login/useStyles.ts

```diff
@@ -1,26 +1,30 @@
 import { makeStyles } from '@material-ui/core/styles';
 
-const useStyles = makeStyles(() => ({
-  root: {
-    minHeight: '100vh',
-    '& .MuiInput-underline:before': {
-      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
-    },
-  },
+const useStyles = makeStyles((theme) => ({
   authWrapper: {
```

Comment:
Most of these styles can be directly applied to the Box component

Eg: <Box display="flex" alignItems="center"...>